flink-java/src/main/java/org/apache/flink/api/java/DataSet.java

```diff
@@ -1377,6 +1377,24 @@ public long count() throws Exception {
 		return new SortPartitionOperator<>(this, field, order, Utils.getCallLocationName());
 	}
 
+	/**
+	 * Locally sorts the partitions of the DataSet on the an extracted key in the specified order.
+	 * DataSet can be sorted on multiple values by returning a tuple from the KeySelector.
+	 *
+	 * Note that any key extraction methods cannot be chained with the KeySelector. To sort the
```

Comment:
"Note that any key extraction methods cannot be ..." -> "Note that no additional sort keys can be appended to a KeySelector."